packages/snap/locales/es.json

```diff
@@ -15,7 +17,7 @@
     "one": "Hay {count} activacion de LXP activa"
   },
   "lxpAddress": {
-    "heading": "Cuenta que usas para Linea XP",
+    "heading": "Cuenta que usas para Linea LXP",
```

Comment:
This is wrong, should be reverted. It's Linea XP.

Reply:
From the discussion we had with the Marketing team, we're going with just "LXP"